server/routes/changeEstablishment.ts

```diff
@@ -59,7 +59,7 @@ export default function routes({ auditService, supportedPrisonsService, userServ
 
     const policyNoticeDaysMin = await supportedPrisonsService.getPolicyNoticeDaysMin(
       res.locals.user.username,
-      res.locals.user.activeCaseLoadId,
+      req.body.establishment,
```

Comment:
This is the bug: on the `POST` to change establishment we should be looking up prison by the newly-selected establishment; **not** the user's current active case load.